ui/tests/integration/components/dashboard/client-count-card-test.js

```diff
@@ -44,24 +39,63 @@ module('Integration | Component | dashboard/client-count-card', function (hooks)
         data: ACTIVITY_RESPONSE_STUB,
       };
     });
+    this.server.get('sys/internal/counters/config', function () {
+      assert.true(true, 'sys/internal/counters/config');
+      return {
+        request_id: 'some-config-id',
+        data: {
+          billing_start_timestamp: LICENSE_START.toISOString(),
+        },
+      };
+    });
 
-    await render(hbs`<Dashboard::ClientCountCard @license={{this.license}} />`);
+    await render(hbs`<Dashboard::ClientCountCard @isEnterprise={{true}} />`);
     assert.dom('[data-test-client-count-title]').hasText('Client count');
-    assert.dom('[data-test-stat-text="total-clients"] .stat-label').hasText('Total');
-    assert
-      .dom('[data-test-stat-text="total-clients"] .stat-text')
-      .hasText('The number of clients in this billing period (Jul 2023 - Jan 2024).');
     assert
-      .dom('[data-test-stat-text="total-clients"] .stat-value')
-      .hasText(`${formatNumber([total.clients])}`);
-    assert.dom('[data-test-stat-text="new-clients"] .stat-label').hasText('New');
-    assert
-      .dom('[data-test-stat-text="new-clients"] .stat-text')
-      .hasText('The number of clients new to Vault in the current month.');
+      .dom(CLIENT_COUNT.statText('Total'))
+      .hasText(
+        `Total The number of clients in this billing period (Jul 2023 - Jan 2024). ${formatNumber([
+          total.clients,
+        ])}`
+      );
+
     assert
-      .dom('[data-test-stat-text="new-clients"] .stat-value')
-      .hasText(`${formatNumber([latestMonth.new_clients.counts.clients])}`);
+      .dom(CLIENT_COUNT.statText('New'))
+      .hasText(
+        `New The number of clients new to Vault in the current month. ${formatNumber([
+          latestMonth.new_clients.counts.clients,
+        ])}`
+      );
+
     // fires second request to /activity
     await click('[data-test-refresh]');
   });
+
+  test('it does not query activity for community edition', async function (assert) {
+    assert.expect(4);
+    this.server.get('sys/internal/counters/activity', () => {
+      // this assertion should NOT be hit in this test
+      assert.true(true, 'uh oh! makes request to sys/internal/counters/activity');
```

Comment:
```suggestion
      throw new Error('uh oh! makes request to sys/internal/counters/activity');
```
^ another way to more clearly verify that the endpoint isn't called in this test.  if you take this approach, you'll also want to adjust the `assert.expect(4)` to `3`

Reply:
Great idea! Much clearer 💡